datasets/de/abgeordnetenwatch/crawler.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+import json
+import math
+from zavod import Context
+from zavod import helpers as h
+
+
+def get_politician_data(context: Context, politician_api_url: str):
+    api_response = context.fetch_json(politician_api_url, cache_days=30)
+    politician = api_response.pop("data")
+    context.audit_data(politician)
+    return politician
+
+
+def crawl(context: Context):
+    # Fetch the source data URL specified in the metadata to a local path:
+    api_response = context.fetch_json(context.dataset.data.url, cache_days=30)
+    total_results = api_response.pop("meta").pop("result").pop("total")
+    num_batches = math.ceil(total_results / 100)
+
+    bi = 0
+    while bi < num_batches:
```

Comment:
no need to change this now, but for future, our pattern is usually

def crawl(context): # loop over the bulk data
    for thing in data:
        crawl_thing(context, thing)

def crawl_think(...): # crawl _and_ emit the thing's data

that keeps the methods a bit smaller

Reply:
just changed it to crawl_politician.